Packs/GCP/Integrations/GCP/GCP.yml

```diff
@@ -0,0 +1,1603 @@
+
+  - name: gcp-compute-subnet-update
+    description: Enables flow logs or Private Google Access on a subnet.
+    arguments:
+    - name: project_id
+      required: true
+      description: GCP project ID.
+    - name: region
+      required: true
+      description: GCP region.
+    - name: resource_name
+      required: true
+      description: Subnet name.
+    - name: enable_flow_logs
+      required: false
+      auto: PREDEFINED
+      predefined:
+      - "true"
+      - "false"
+      description: Enable VPC Flow Logs.
+    - name: enable_private_ip_google_access
+      required: false
+      auto: PREDEFINED
+      predefined:
+      - "true"
+      - "false"
+      description: Enable Private Google Access.
+    outputs:
+    - contextPath: GCP.Compute.Operations.name
+      description: The name of the updated subnet.
+      type: String
+    - contextPath: GCP.Compute.Operations.region
+      description: The region of the updated subnet.
+      type: String
+    - contextPath: GCP.Compute.Operations.subnetName
+      description: The name of the subnet that was updated.
+      type: String
+    - contextPath: GCP.Compute.Operations.enableFlowLogs
+      description: Whether flow logs are enabled for the subnet.
+      type: Boolean
+    - contextPath: GCP.Compute.Operations.ipCidrRange
+      description: The updated CIDR range for the subnet.
+      type: String
+    - contextPath: GCP.Compute.Operations.privateIpGoogleAccess
+      description: Whether private Google access is enabled for the subnet.
+      type: Boolean
+    - contextPath: GCP.Compute.Operations.privateIpv6GoogleAccess
+      description: Whether private IPv6 Google access is enabled for the subnet.
+      type: Boolean
+    - contextPath: GCP.Compute.Operations.stackType
+      description: The stack type of the subnet (e.g., `IPV4`, `IPV6`).
+      type: String
+    - contextPath: GCP.Compute.Operations.rangeName
+      description: The name of the IP range associated with the subnet.
+      type: String
+    - contextPath: GCP.Compute.Operations.secondaryIpRanges
+      description: List of secondary IP ranges for the subnet.
+      type: Unknown
+    - contextPath: GCP.Compute.Operations.description
+      description: The description of the subnet.
+      type: String
+
+  - name: gcp-compute-instance-metadata-add
+    description: Sets metadata for the specified instance to the data included in the request.
+    arguments:
+    - name: project_id
+      required: true
+      description: GCP project ID.
+    - name: zone
+      required: true
+      description: The name of the zone for this request.
+    - name: resource_name
+      required: true
+      description: Name of the instance scoping this request.
+    - name: metadata
+      required: true
+      description: 'Metadata to be made available to the guest operating system running on the instances. Each entry is a key/value pair separated by '';'' like so: key=abc,value=123;key=abc,value=123.'
```

Comment:
Metadata to be made available to the guest operating system running on the instances. Each entry is a key/value pair separated by '';''. For example, key=abc,value=123;key=abc,value=123